### ISSUE TITLE
Client and Site Columns in Audit Log

### ISSUE THREAD

**Issue by factulas** [amidaware/tacticalrmm#827]
Currently I use the Client field for service levels . OS Support/ Application Support/ POS - Support and would like that to Show in the audit log for Clients (listed in site)

Regardless the audit log does not show Client or Site information, which, I believe would be beneficial being shown. If not for the way I use it but even the intentional use case of Client and Site Audit as many Hostnames are the same ie. Mainserver through all clients and sites. 

TLDR; 
Would like Client and Site fields in audit log as it's hard to tell Clients apart. 

Thank You,
Jay

**Comment by sadnub** [comment]
This was implemented in the last release. Thanks!
